kernel/mmanutils.c

```diff
@@ -751,6 +766,9 @@ long myst_mmap(
         ECHECK(
             myst_mman_mmap(&_mman, addr, length, prot, tflags, (void**)&ret));
 
+        //Update mem usage myst_mmap
```

Comment:
As max usage is only displayed if `--perf` was passed on command line, its better to apply the same condition here.